File: consumer/src/main/java/com/glancebar/example/consumer/ConsumerApplication.java
package com.glancebar.example.consumer;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ImportResource;

/**
 * @author YISHEN CAI
 */
@SpringBootApplication
@ImportResource(value = { "classpath:dubbo/dubbo-consumer.xml" })
public class ConsumerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ConsumerApplication.class);
    }
}

File: consumer/src/main/kotlin/com/glancebar/example/consumer/ConsumerApplication.kt
package com.glancebar.example.consumer

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ImportResource

@SpringBootApplication
@ImportResource(value = [
    "classpath:dubbo/dubbo-consumer.xml"
])
class ConsumerApplication

fun main(args: Array<String>) {
    runApplication<ConsumerApplication>(*args)
}
